// exception.cpp : Defines the entry point for the console application.
//

#include "stdafx.h"
#include<iostream>

#include<iomanip>
#include<sstream>
#include<ctime>
using namespace std;
int main()
{
	try
	{
		//throw 20;
		throw "error";
	}
	catch(int e)
	{
		cout<<"exception occured"<<e<<"\n";
	}
	catch(const char*const c_error )
	{
		cerr<<"exception occured"<<c_error<<"\n";
	}
	return 0;
}

